### ISSUE TITLE
[ADD] isinstance-basestring: If isinstance(var, (str, unicode)) then emit message to use isinstance(var, basestring)

### ISSUE THREAD

**Issue by moylop260** [PyCQA/pylint#995]
### Steps to reproduce
```bash
$ cat a.py
# isinstance(var, (str, unicode))
$ pylint a.py
```

### Current behavior
Check not exists

### Expected behavior
Emit message to use isinstance(var, basestring)
![screen shot 2016-07-11 at 10 28 54 am](https://cloud.githubusercontent.com/assets/6644187/16737182/c8f4c732-4755-11e6-96cb-1a11c6f3b8b0.png)

NOTE: I can help to create a PR with this check, just tell me if is valid for you and if is a global check or extension...

**Comment by PCManticore** [comment]
This is true for Python 2 only. It does not make sense to have this check into Pylint.

**Comment by moylop260** [comment]
Yes, you are right
I missed comment the use of `max-version`

**Comment by moylop260** [comment]
Do make sense for you if we use `maxversion: (2, 7)` and a extensions or definitively we don't like this one in pylint?

**Comment by PCManticore** [comment]
This can be a plugin in your projects, it doesn't have to live in Pylint. Given the fact we plan to drop support for Python 2 in two years, it is not worth including extensions that are aiming it in particular. The only exception is the Python 3 porting checker, but that is a completely different topic.

**Comment by moylop260** [comment]
Thanks for reply
I got it
